packages/base-map/i18n/en-US.yml

```diff
@@ -0,0 +1,16 @@
+otpUi:
+  mapControls:
+    CooperativeGesturesHandler:
+      MacHelpText: Use ⌘ + scroll to zoom the map
+      MobileHelpText: Use two fingers to move the map
+      WindowsHelpText: Use Ctrl + scroll to zoom the map
+    NavigationControl:
+      ResetBearing: Reset bearing to north
+      ZoomIn: Zoom in
+      ZoomOut: Zoom out
+    ScaleControl:
```

Comment:
Where does the scale control get used? I don't see a scale on most maps.

Reply:
This is in the "with optional controls" story, but we can always remove that scale from the map if it's not representative of how we use it?